auctions/models.py

```diff
@@ -528,27 +529,51 @@ def __unicode__(self):
     def fees(self):
         return PayoutFee.objects.filter(payout=self)
 
+    def credits(self):
+        return PayoutCredit.objects.filter(payout=self)
+
     def request(self):
         receiver = (
             PAYPAL_PAYOUT_RECIPIENT if PAYPAL_PAYOUT_RECIPIENT
             else self.claim.user.email
         )
+
+        total_payout_amount = self.amount
+
         paypal_fee = PayoutFee(
             payout=self,
             fee_type='PayPal',
             amount=Decimal('0.25')
         )
         paypal_fee.save()
 
+        try:
+            user_offers = Offer.objects.filter(user=self.claim.user)
+            if user_offers:
+                total_refund = (
+                    user_offers.aggregate(Sum('amount'))['amount__sum']
+                )
+                refund_user_offer = PayoutCredit(
+                    payout=self,
+                    fee_type='refund',
+                    description="Your offer",
+                    amount=total_refund,
+                )
+                refund_user_offer.save()
+                total_payout_amount += total_refund
+
+        except Offer.DoesNotExist:
+            refund_user_offer = None
```

Comment:
This will probably conflict with any other changes going into `Payout.request` ... merge this first?